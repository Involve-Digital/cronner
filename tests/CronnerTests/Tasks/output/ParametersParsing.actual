array(
	'cronner-task' => 'Bileto\\Cronner\\tests\\objects\\TestObject - test01',
	'cronner-period' => '5 minutes',
	'cronner-days' => NULL,
	'cronner-days-of-month' => NULL,
	'cronner-time' => NULL,
)
