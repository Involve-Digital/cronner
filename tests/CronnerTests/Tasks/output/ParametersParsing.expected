array(
	'cronner-task' => 'E-mail notifications',
	'cronner-period' => '5 minutes',
	'cronner-days' => NULL,
	'cronner-days-of-month' => NULL,
	'cronner-time' => NULL,
)
